examples/connector/python/connector/main.py

```diff
@@ -0,0 +1,176 @@
+import grpc
+from concurrent import futures
+import json
+import datetime
+
+
+import connector_sdk_pb2_grpc as connector__sdk__pb2__grpc
```

Comment:
why are we importing in this manner, using double `_`?

Reply:
It is done to keep consistent to what generated files are using. I guess we can directly import as well.